addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChatWindow" owl="1">
-        <div class="o_ChatWindow" tabindex="0"
+        <div class="o_ChatWindow" tabindex="0" t-if="chatWindow" t-att-data-visible-index="chatWindow.visibleIndex"
```

Comment:
No `t-if`, this condition sets or not the root not, which is not what is intended here.
You should guard read of `chatWindow` fields in this part of the template:
`t-att-data-visible-index="chatWindow ? chatWindow.visibleIndex : undefined"`